src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final String defaultColor = "WHITE";
+    private final String defaultName = "circle";
+    private final int defaultRadius = 10;
```

Comment:
Looks like constants. Make it static and follow [constants name convention](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names).